lib/src/widgets/layout/flex/separated_column.dart

```diff
@@ -24,17 +25,21 @@ class SeparatedColumn extends StatelessWidget {
   final VerticalDirection verticalDirection;
   final TextBaseline? textBaseline;
   final Clip clipBehavior;
+  final EdgeInsetsGeometry? childPadding;
 
   @override
   Widget build(BuildContext context) {
+    final List<Widget> paddedChildren =
+        children.map((child) => childPadding == null ? child : Padding(padding: childPadding!, child: child)).toList();
```

Comment:
```suggestion
        children.map((child) => childPadding == null ? child : Padding(padding: childPadding!, child: child));
```

I think the toList is not needed

Reply:
fixed